packages/account/src/providers/memory-cache.test.ts

```diff
@@ -1,11 +1,8 @@
 import { randomBytes } from '@fuel-ts/crypto';
 import type { BytesLike } from '@fuel-ts/interfaces';
-import { hexlify } from '@fuel-ts/utils';
 
 import { MemoryCache } from './memory-cache';
 
-const CACHE_ITEMS = [hexlify(randomBytes(8)), randomBytes(8), randomBytes(8)];
```

Comment:
`CACHE_ITEMS[0]` is hexlified and was testing that `MemoryCache` works with string keys as well. Can you please add one more test that uses `hexlify`?

```ts
  it('can set (string)', () => {
    const ttl = 1000;
    const expiresAt = Date.now() + ttl;
    const memCache = new MemoryCache(ttl);
    const value = hexlify(randomBytes(8));

    expect(memCache.set(value)).toBeGreaterThanOrEqual(expiresAt);
  });
```

Reply:
🚢 ?